squarelet/organizations/querysets.py

```diff
@@ -45,6 +45,11 @@ def create_individual(self, user):
         return user.individual_organization
 
 
+class MembershipQuerySet(models.QuerySet):
+    def get_viewable(self, user):
+        return user.memberships.filter(Q(private=False) | Q(organization__users=self.request.user))
```

Comment:
We should add a test to this - I see I have not done a good job at testing querysets explicitly in the past, but might be a good time to add them.

I know you took this code from my comment, but that was written in the context of a view and not a queryset, and needs to be adapted to work like this.  You may want to read up on QuerySets: https://docs.djangoproject.com/en/3.0/ref/models/querysets/

1) The `get_viewable` method should return all viewable memberships for the user, not just memberships for that user  - so it should be `return self.filter(...` not `return user.memberships.filter(...` as in the view, where we only want memberships for the user we have specified in the URL.  So in the view, with this method, it would be something along the lines of `user.memberships.get_viewable(self.request.user)`

2) `private=False` is on the Organization, not the membership, so it should be `organization__private=False`

3) `Q(organization__users=self.request.user)` should be `Q(organization__users=user)` - we want memberships that the passed in user are a part of.  `self` here is the queryset, which does not have a request attribute - that only exists at the view level.

Reply:
Got it - and I read through most of the QuerySet documentation. I also reviewed the viewset docs again so I'm clear. I made these changes, and I'll now take a stab at writing tests for it.

Reply:
I've added tests for the queryset directly (new file: test_querysets.py) and the expected api response (test_api.py)